compiler/testdata/coroutine.go

```diff
@@ -577,3 +577,30 @@ func IdentityGenericInt(n int) {
 func IdentityGenericStructInt(n int) {
 	(&IdentityGenericStruct[int]{n: n}).Run()
 }
+
+type Box struct {
+	x int
+}
+
+func (b *Box) Closure(y int) func(int) {
+	// Force compilation of this method and the closure within.
+	// Remove once #84 is fixed.
```

Comment:
We're starting to pile so many TODOs on issue #84 it's almost becoming famous :D